core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/ControllerEvent.scala

```diff
@@ -35,4 +37,5 @@ object ControllerEvent {
       tuple: Array[(ITuple, ActorVirtualIdentity)]
   ) extends ControlCommand[Unit]
 
+  case class WorkerAssigned(workerMapping: Map[String, Seq[String]]) extends ControlCommand[Unit]
```

Comment:
seems a weird name to me, how about `AssignWorkers`, or `WorkerAssignment`

Reply:
hmm, all the other ControlEvents are `xxxTriggered` or `xxxUpdate`, maybe I can use `WorkerAssignmentUpdate`?

Reply:
changed to `WorkerAssignmentUpdate`.